cmd/webhook/main.go

```diff
@@ -79,7 +79,7 @@ func main() {
 	configMapWatcher := configmap.NewInformedWatcher(kubeClient, system.Namespace)
 	configMapWatcher.Watch(logging.ConfigName, logging.UpdateLevelFromConfigMap(logger, atomicLevel, component))
 	if err = configMapWatcher.Start(stopCh); err != nil {
-		logger.Fatalf("failed to start configuration manager: %v", err)
+		logger.Fatal("Failed to start the ConfigMap watcher", zap.Error(err))
```

Comment:
Should this be [`Fatalw`](https://godoc.org/go.uber.org/zap#SugaredLogger.Fatalw)?

Reply:
I am still new using Zap; however, my understanding is that we don't need Fatalw as err is surrounded in zap.Error() which creates the key/value structure for us. The equivalent if we were using Fatalw would be:

logger.Fatalw("Failed to start the ConfigMap wacher",
"error", err)

The output for either should look like:

{"level":"error", "msg":"Failed to start the ConfigMap watcher", "error":"<error information>"}

Reply:
> Fatal uses fmt.Sprint to construct and log a message, then calls os.Exit.

> Sprint formats using the default formats for its operands and returns the resulting string. Spaces are added between operands when neither is a string.

So I think this would produce:

{"level":"error", "msg":"Failed to start the ConfigMap watcher {\"error\": \"\"}"}

You may be right on the keys and values separate.

Have you been able to test the output here?

Reply:
So went back and tested the output and you are correct here. The example in the documentation ( https://godoc.org/go.uber.org/zap ) was a bit misleading. 

It looks like we have this same mistake in other parts of the code such as in activator. I have made the fix here. I will make a new PR to fix the other locations.